API.md

```diff
@@ -3,55 +3,60 @@
 To use Draw
 
 ```js
-var Draw = new MapboxDraw({ options });
-map.addControl(Draw);
+// Create a Mapbox GL JS map
+var map = new Map(mapOptions);
+
+// Create a Draw control
+var draw = new MapboxDraw(drawOptions);
+
+// Add the Draw control to your map
+map.addControl(draw);
 ```
 
-Draw only works after the map has loaded so it is wise to perform any interactions in the `load` event callback of mapbox-gl:
+**Draw only works after the Mapbox GL JS map has loaded**, so it is wise to perform any interactions after your maps `load` event:
```

Comment:
> Should we go so far as saying required rather than wise?

Yes.

> Should we make all the Draw functions throw errors if load isn't done?

I think that would be a good idea, yeah.